### PR TITLE
Issue 74 model upload

### LINKED ISSUE #74: Model app
Create an application to upload and view models in the project

### PR DESCRIPTION
**Pull request by martinjrobins**
Fixes #74 

Adds a file field to pkpd_model add view, so you can upload a file (the update view is the same)
adds urls for:
- adding a new pkpd_model
- adding a new pkpd_model and adding this to a given project

adds link in project detail view to add a new pkpd_model and add it to this project

NOTE: this is branched off from #89, so #89 must be merged before this one!!!

### CODE REVIEW

File: pkpdapp/pkpdapp/forms.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        if 'project' in kwargs:
+            self.project_id = kwargs.pop('project')
+        else:
+            self.project_id = None
+
+        super().__init__(*args, **kwargs)
+
+    class Meta:
+        model = PkpdModel
+        fields = ('name', 'description', 'model_type', 'sbml')
```
Comment (ben18785) [inline pkpdapp/pkpdapp/forms.py:100]: Can we make the description field optional? I guess we do that my changing the PkpdModel class?

**Review by ben18785 — CHANGES_REQUESTED** [review]
I checked this with an SBML upload file and another that wasn't XML: all seems dandy. A couple of really minor points but, otherwise, looks good (and I've learned a lot from reading it).

File: pkpdapp/pkpdapp/forms.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        widget=forms.Textarea(attrs={'rows': 2, 'cols': 25}),
+        label='Description',
+        required=False
+    )
+
+
+class CreateNewPkpdModel(forms.ModelForm):
+    """
+    A form to create a new :model:`pkpdapp.PkpdModel`, which allows a user to
+    upload their sbml from a file.
```
Comment (ben18785) [inline pkpdapp/pkpdapp/forms.py:85]: In some places, (e.g. in the model definition) SBML opposed to sbml is used...

File: pkpdapp/pkpdapp/forms.py (hunk 1 above)
Comment (martinjrobins) [reply pkpdapp/pkpdapp/forms.py:100]: yup, done this

File: pkpdapp/pkpdapp/forms.py (hunk 2 above)
Comment (martinjrobins) [reply pkpdapp/pkpdapp/forms.py:85]: updated to SBML

**Pull request merged**